unit/models/check_stop_payment.py

```diff
@@ -0,0 +1,33 @@
+from unit.models import *
+from unit.utils import date_utils
+
+
+StopPaymentStatus = Literal["Active", "Disabled"]
+
+class CheckStopPaymentDTO(object):
+    def __init__(self, id: str, created_at: datetime, updated_at: datetime, amount: int, status: StopPaymentStatus,
+                 check_number: str, tags: Optional[Dict[str, str]], relationships: Optional[Dict[str, Relationship]]):
+        self.id = id
+        self.type = "checkStopPayment"
```

Comment:
This might need to be just `stopPayment`

Reply:
oh? I saw `checkStopPayment` in the docs

Reply:
I thought I saw that it was inconsistent in the examples in the docs but I can't recall where I saw it.  But yeah, I think it is likely `checkStopPayment` too.